src/gui/folder.cpp

```diff
@@ -769,6 +769,10 @@ void Folder::setVirtualFilesEnabled(bool enabled)
     }
 
     if (newMode != _definition.virtualFilesMode) {
+        // TODO: Must wait for current sync to finish!
```

Comment:
Is this still a TODO? I think we can safely say now that it's up to the caller to prevent this.

Reply:
Still a TODO, removing // TODO would not solve anything.
For the next major release I'll provide a solution.